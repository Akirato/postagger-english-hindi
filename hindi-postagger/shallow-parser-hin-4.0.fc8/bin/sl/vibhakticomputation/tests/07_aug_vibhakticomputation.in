<Sentence id="1">
1	((	NP	<fs af='ਪਹਾੜ,n,m,pl,3,o,,' head=ਪਹਾੜਾਂ>
1.1	ਸਿਰਫ	JJ	<fs af='ਸਿਰਫ,adj,,,,,,' name=ਸਿਰਫ>
1.2	ਪਹਾੜਾਂ	NN	<fs af='ਪਹਾੜ,n,m,pl,3,o,,' name=ਪਹਾੜਾਂ>
1.3	ਦੀ	PSP	<fs af='ਦਾ,psp,f,sg,,d,,' name=ਦੀ>
	))		
2	((	NP	<fs af='ਦੁਨੀਆਂ,n,f,sg,3,o,,' head=ਦੁਨੀਆਂ>
2.1	ਦੁਨੀਆਂ	NN	<fs af='ਦੁਨੀਆਂ,n,f,sg,3,o,,' name=ਦੁਨੀਆਂ>
2.2	ਤੋਂ	PSP	<fs af='ਤੋਂ,psp,,,,,,' name=ਤੋਂ>
2.3	ਇਲਾਵਾ	NEG	<fs af='ਇਲਾਵਾ,psp,,,,,,' name=ਇਲਾਵਾ poslcat=NM>
	))		
3	((	NP	<fs af='ਰਸਤਾ,n,m,sg,3,o,,' head=ਰਸਤੇ>
3.1	ਰਸਤੇ	NN	<fs af='ਰਸਤਾ,n,m,sg,3,o,,' name=ਰਸਤੇ>
3.2	ਵਿੱਚ	PSP	<fs af='ਵਿੱਚ,psp,any,sg,,d,,' name=ਵਿੱਚ>
	))		
4	((	NP	<fs af='ਇਨਸਾਨ,n,m,sg,3,d,,' head=ਇਨਸਾਨ>
4.1	ਇਨਸਾਨ	NN	<fs af='ਇਨਸਾਨ,n,m,sg,3,d,,' name=ਇਨਸਾਨ>
	))		
5	((	CCP	<fs af='ਤਾਂ,avy,,,,,,' head=ਤਾਂ>
5.1	ਤਾਂ	CC	<fs af='ਤਾਂ,avy,,,,,,' name=ਤਾਂ>
	))		
6	((	RBP	<fs af='ਕੀ,pn,any,sg,,d,,' head=ਕੀ>
6.1	ਕੀ	WQ	<fs af='ਕੀ,pn,any,sg,,d,,' name=ਕੀ>
	))		
7	((	NP	<fs af='ਪਰਿੰਦਾ,n,m,sg,3,d,,' head=ਪਰਿੰਦਾ>
7.1	ਕੋਈ	PRP	<fs af='ਕੋਈ,pn,any,sg,,d,,' name=ਕੋਈ>
7.2	ਪਰਿੰਦਾ	NN	<fs af='ਪਰਿੰਦਾ,n,m,sg,3,d,,' name=ਪਰਿੰਦਾ>
7.3	ਵੀ	RP	<fs af='ਵੀ,avy,,,,,,' name=ਵੀ>
	))		
8	((	VGF	<fs af='ਦਿਸ,v,m,sg,any,,ਦਾ,ਦਾ' head=ਦਿਸਦਾ>
8.1	ਨਹੀਂ	NEG	<fs af='ਨਹੀਂ,avy,,,,,,' name=ਨਹੀਂ>
8.2	ਦਿਸਦਾ	VM	<fs af='ਦਿਸ,v,m,sg,any,,ਦਾ,ਦਾ' name=ਦਿਸਦਾ>
8.3	.	SYM	<fs af='&sdot;,punc,,,,,,' name=.>
	))		
</Sentence>
