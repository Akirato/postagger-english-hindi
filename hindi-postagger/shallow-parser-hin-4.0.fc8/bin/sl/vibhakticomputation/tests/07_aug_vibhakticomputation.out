<Sentence id="1">
1	((	NP	<fs af='ਪਹਾੜ,n,m,pl,3,o,0_ਦਾ,' head=ਪਹਾੜਾਂ vpos="vib2_3">
1.1	ਸਿਰਫ	JJ	<fs af='ਸਿਰਫ,adj,,,,,,' name=ਸਿਰਫ>
1.2	ਪਹਾੜਾਂ	NN	<fs af='ਪਹਾੜ,n,m,pl,3,o,,' name=ਪਹਾੜਾਂ>
	))		
2	((	NP	<fs af='ਦੁਨੀਆਂ,n,f,sg,3,o,0_ਤੋਂ,' head=ਦੁਨੀਆਂ vpos="vib1_2">
2.1	ਦੁਨੀਆਂ	NN	<fs af='ਦੁਨੀਆਂ,n,f,sg,3,o,,' name=ਦੁਨੀਆਂ>
2.2	ਇਲਾਵਾ	NEG	<fs af='ਇਲਾਵਾ,psp,,,,,,' name=ਇਲਾਵਾ poslcat=NM>
	))		
3	((	NP	<fs af='ਰਸਤਾ,n,m,sg,3,o,0_ਵਿੱਚ,' head=ਰਸਤੇ vpos="vib1_2">
3.1	ਰਸਤੇ	NN	<fs af='ਰਸਤਾ,n,m,sg,3,o,,' name=ਰਸਤੇ>
	))		
4	((	NP	<fs af='ਇਨਸਾਨ,n,m,sg,3,d,,' head=ਇਨਸਾਨ>
4.1	ਇਨਸਾਨ	NN	<fs af='ਇਨਸਾਨ,n,m,sg,3,d,,' name=ਇਨਸਾਨ>
	))		
5	((	CCP	<fs af='ਤਾਂ,avy,,,,,,' head=ਤਾਂ>
5.1	ਤਾਂ	CC	<fs af='ਤਾਂ,avy,,,,,,' name=ਤਾਂ>
	))		
6	((	RBP	<fs af='ਕੀ,pn,any,sg,,d,,' head=ਕੀ>
6.1	ਕੀ	WQ	<fs af='ਕੀ,pn,any,sg,,d,,' name=ਕੀ>
	))		
7	((	NP	<fs af='ਪਰਿੰਦਾ,n,m,sg,3,d,,' head=ਪਰਿੰਦਾ>
7.1	ਕੋਈ	PRP	<fs af='ਕੋਈ,pn,any,sg,,d,,' name=ਕੋਈ>
7.2	ਪਰਿੰਦਾ	NN	<fs af='ਪਰਿੰਦਾ,n,m,sg,3,d,,' name=ਪਰਿੰਦਾ>
7.3	ਵੀ	RP	<fs af='ਵੀ,avy,,,,,,' name=ਵੀ>
	))		
8	((	VGF	<fs af='ਦਿਸ,v,m,sg,any,,ਦਾ,ਦਾ' head=ਦਿਸਦਾ>
8.1	ਨਹੀਂ	NEG	<fs af='ਨਹੀਂ,avy,,,,,,' name=ਨਹੀਂ>
8.2	ਦਿਸਦਾ	VM	<fs af='ਦਿਸ,v,m,sg,any,,ਦਾ,ਦਾ' name=ਦਿਸਦਾ>
8.3	.	SYM	<fs af='&sdot;,punc,,,,,,' name=.>
	))		
</Sentence>
